cmd/gateway/azure/gateway-azure.go

```diff
@@ -625,8 +625,17 @@ func (a *azureObjects) ListBuckets(ctx context.Context) (buckets []minio.BucketI
 // DeleteBucket - delete a container on azure, uses Azure equivalent `ContainerURL.Delete`.
 func (a *azureObjects) DeleteBucket(ctx context.Context, bucket string, forceDelete bool) error {
 	containerURL := a.client.NewContainerURL(bucket)
-	_, err := containerURL.Delete(ctx, azblob.ContainerAccessConditions{})
-	return azureToObjectError(err, bucket)
+
```

Comment:
DeleteBucket() has forceDelete flag, so we need to return BucketNotEmpty only when forceDelete is set to true.

Reply:
@vadmeste means when forcedelete is set to false

Reply:
will do that change?